cluster/gce/gci/flexvolume_node_setup.sh

```diff
@@ -99,31 +99,31 @@ generate_chroot_wrapper() {
     exit 1
   fi
 
-  for driver_dir in ${MOUNTER_PATH}/flexvolume/*; do
+  for driver_dir in "${MOUNTER_PATH}/flexvolume"/*; do
     if [ -d "$driver_dir" ]; then
 
-      filecount=$(ls -1 $driver_dir | wc -l)
-      if [ $filecount -gt 1 ]; then
+      filecount=$(cd "$driver_dir"; find . -mindepth 1 -maxdepth 1 -print0 | xargs -0 -n1 basename | grep -c -v "^\.")
```

Comment:
we shouldn't need to do grep after the basename?

Reply:
This line counts files under $driver_dir. Therefore, I think need `grep -c` to count.

Reply:
we can count with something like wc

Reply:
I considered using `wc -l`, but shellcheck said SC2126.
```
... xargs -0 -n1 basename | grep -v "^\." | wc -l)
                            ^-----------^ SC2126: Consider using grep -c instead of grep|wc -l.
```

Reply:
like I said up top for both of these now-find-based-implementations, I feel like the original ls-based implementations are fine; it's not clear to me what value we're getting here by adding all of this extra complexity to work around corner cases that haven't crept up thus far

Reply:
er right, see discussion below for if we even need to ignore files starting with `.` (I don't think so, however we probably _should_ ignore directories)